components/popup/overview.md

```diff
@@ -0,0 +1,130 @@
+
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+| Parameter | Type | Description |
+| ----------- | ----------- | ----------- |
+| `AnchorHorizontalAlign` | `PopupAnchorHorizontalAlign` enum <br /> (`Left`) | Defines how the anchor aligns with the Popup component on the horizontal plane. [Read more about Popup Positioning.]({%slug popup-position-collision%})|
+| `AnchorSelector` | `string` | Specifies the CSS selector targeting the element that the Popup uses as an anchor. |
+| `AnchorVerticalAlign` | `PopupAnchorVerticalAlign` enum <br /> (`Bottom`) | Defines how the anchor aligns with the Popup on the vertical plane. [Read more about Popup Positioning.]({%slug popup-position-collision%}). |
+| `AnimationDuration` | `int` | Defines the duration of the animation in milliseconds. [Read more about Popup animations.]({%slug popup-animation%}) |
+| `AnimationType` | `AnimationType` enum <br /> (`SlideDown`) | Sets the type of animation when the component displays and hides. [Read more about Popup animations.]({%slug popup-animation%}) |
+| `HorizontalAlign` | `PopupHorizontalAlign ` enum <br /> (`Left`) | Determines if the left or the right side of the Popup will touch its anchor. [Read more about Popup Positioning.]({%slug popup-position-collision%}) |
+| `HorizontalCollision` | `PopupCollision  ` enum <br /> (`Fit`) | Sets the behavior of the Popup when it doesn't fit in the viewport based on the horizontal plane. [Read more about Popup collision behavior.]({%slug popup-position-collision%}) |
+| `HorizontalOffset` | `double` | Sets the horizontal space between the Popup and its anchor in `pixels`. |
+| `VerticalAlign` | `PopupVerticalAlign ` enum <br /> (`Top`) | Determines if the Popup will touch the anchor with its top, bottom, or center. [Read more about Popup Positioning.]({%slug popup-position-collision%}) |
+| `VerticalCollision` | `PopupCollision  ` enum <br /> (`Flip`) | Sets the behavior of the Popup when it doesn't fit in the viewport based on the vertical plane. [Read more about Popup collision behavior.]({%slug popup-position-collision%}) |
+| `VerticalOffset` | `double` | Sets the vertical space between the Popup and its anchor in `pixels`. |
+
+### Styling and Appearance
+
+The following parameters enable you to customize the appearance of the Blazor Popup:
+
+| Parameter | Type | Description |
+| --- | --- | --- |
+| `Class` | `string` | The custom CSS class to be rendered on the `<div>` element, which wraps the component `ChildContent`. Use for [styling customizations]({%slug themes-override%}). |
+| `Height` | `string` | The height of the Popup. |
+| `Width` | `string` | The width of the Popup. |
+
+## Popup Reference and Methods
+
+To execute Popup methods, obtain a reference to the component instance with `@ref`.
+
+| Method  | Description |
+|---------|-------------|
+| `Refresh` | Use this method to programmatically re-render the Popup. |
```

Comment:
Provide more context and tips. See https://docs.telerik.com/blazor-ui/components/dialog/overview#dialog-reference-and-methods